client/src/main/java/client/Client.java

```diff
@@ -0,0 +1,63 @@
+package client;
+
+
+import java.io.*;
+import java.net.Socket;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class Client {
+
+    private static final String SERVER_HOST = "localhost";
+    private static final int SERVER_PORT = 8080;
+
+    private Socket socket;
+    private BufferedReader reader;
+    private PrintWriter writer;
+
+    public void connect() {
+        try {
+            socket = new Socket(SERVER_HOST, SERVER_PORT);
+            reader = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+            writer = new PrintWriter(socket.getOutputStream(), true);
+            startMessageReader();
+        } catch (IOException e) {
+            Logger logger = Logger.getLogger(Client.class.getName());
```

Comment:
would be better to get the logger on the class level and then reuse it in methods